src/include/sof/audio/kpb.h

```diff
@@ -99,6 +105,7 @@ struct dd {
 	uint8_t is_draining_active;
 	enum kpb_state *state;
 	size_t sample_width;
+	size_t buffered_while_draining;
```

Comment:
Is it read anywhere? Because in this patch I just saw it being only written to.

Reply:
New commits were pushed, there is the answer.